conf/prod.routes

```diff
@@ -3,6 +3,9 @@
 ->         /setup                   setup.Routes
 ->         /email                   email.Routes
 ->         /secure-message          secureMessage.Routes
+->         /secure-comms-alert      secureCommsAlert.Routes
+->         /hmrc/email              email.Routes
```

Comment:
If `/hmrc/email` is the new route prefix for `email.Routes` then I'd remove the other one to avoid confusion.